packages/react-slider/MIGRATION.md

```diff
@@ -1,58 +1,30 @@
 # Slider Migration
 
-## STATUS: WIP 🚧
-
-This Migration guide is a work in progress and is not yet ready for use.
+The Slider control is a thin wrapper around an `<input type="range"/>`. Additional elements like labels and current value will need to be explicitly rendered.
 
 ## Migration from v8
 
-- `componentRef` => NOT SUPPORTED - use regular `ref` instead.
-- `styles` => NOT SUPPORTED - use new styling system via `tokens` instead.
-- `theme` => NOT SUPPORTED
-- `originFromZero` => `origin` - allows for custom origin values other than zero.
-- `ariaLabel` => NOT SUPPORTED
-- `valueLabel` => NOT SUPPORTED
-- `defaultLowerValue` => NOT SUPPORTED
-- `lowerValue` => NOT SUPPORTED
-- `showValue` => NOT SUPPORTED
-- `onChanged` => NOT SUPPORTED
-- `snapToStep` => NOT SUPPORTED - the slider no longer snaps.
-- `buttonProps` => NOT SUPPORTED
-- `valueFormat` => NOT SUPPORTED
-- `ranged` => NOT SUPPORTED
-- `label` => NOT SUPPORTED
+- `componentRef` => use `ref` instead.
+- `styles` => pass classNames to individual slots
+- `theme` => use `FluentProvider` HOC instead
+- `label` => Use `Label` control with `htmlFor` and `id`
+  <<<<<<< HEAD
```

Comment:
Seems like you have some mere conflict issues here.

Reply:
doh